### ISSUE TITLE
Option to send already selected URIs

### ISSUE THREAD

**Issue by Swati4star** [zhihu/Matisse#472]
Is there an option to send already selected Uris to the Mattise activity, so that those images are shown checked already?

**Comment by REBOOTERS** [comment]
no such feature right now . 

what's the point of this feature ? the selected images are shown  checked already ,when click sure button ,those images will be selected once again .which might  not be a good idea .

**Comment by Swati4star** [comment]
Let me give you a use case for this feature,

I go to Matisse activity, select few images. Then, select sure.
Now, I might want to add more images to my selection. 

It will be good idea to show the previously selected images already in the Matisse activity. That way, user will know which images he selected previously.

**Comment by Swati4star** [comment]
This library : https://github.com/ParkSangGwon/TedPicker

also allows you to send selected images uri.

```
Intent intent = new Intent(mActivity, ImagePickerActivity.class);
intent.putExtra(ImagePickerActivity.EXTRA_IMAGE_URIS, uris);
```

**Comment by REBOOTERS** [comment]
how to distinguish previous selected and new selected images when I press sure button ? any good idea ?

**Comment by Swati4star** [comment]
There won't be any differentiation.

It's just that, library should provide a parameter to already select few Uris. The only change in UI would be that, instead of zero items shown checked at first, it will be pre filled with the (optional) parameter passed.

**Comment by Anzhelka** [comment]
This is a very cool feature!

**Comment by JaydipRadadiya** [comment]
Hey, 
Did anyone find the solution? Need help!

**Comment by Sh1d0w** [comment]
This will be a nice feature which will unblock another issue on my flutter_multi_image_picker plugin which uses Matisse https://github.com/Sh1d0w/multi_image_picker/issues/60

Hope it can be implemented soon.

**Comment by LuilliCapellan** [comment]
So, is this going to be implemented anytime soon?
